frontend/src/containers/Faq.js

```diff
@@ -93,24 +93,41 @@ export class Faq extends Component {
-              <li><span>$5k</span> <a href="https://www.linkedin.com/in/xaviercorman">Xavier Corman</a> (<a href="http://edebex.com">Edebex</a>)</li>
+              <li><span>$250k</span> <a href="http://generalcatalyst.com">General Catalyst</a> (SF/NYC/Boston, <a href="https://www.linkedin.com/in/hemanttaneja">Hemant Taneja</a>)</li>
+              <li><span>$50k</span> <a href="https://www.linkedin.com/in/jsiegel">Jonathan Siegel</a> (can't pin down his location)</li>
+              <li><span>$50k</span> <a href="http://belcube.com">Belcube</a> (Brussels)</li>
+              <li><span>$50k</span> <a href="https://www.linkedin.com/in/teljamou">Tony Jamous</a> (London, <a href="https://nexmo.com">Nexmo</a>)</li>
+              <li><span>$25k</span> <a href="https://uk.linkedin.com/in/enadalin">Eric Nadalin</a> (London, <a href="https://nexmo.com">Nexmo</a>)</li>
+              <li><span>$25k</span> <a href="https://www.linkedin.com/in/brian-larson-43904010">Brian Larson</a> (SF, engineer at Google, Twitter)</li>
+              <li><span>$15k</span> <a href="https://twitter.com/rauchg">Guillermo Rauch</a> (SF/Buenos Aires, <a href="http://socket.io">socket.io</a>)</li>
+              <li><span>$10k</span> <a href="http://buytaert.net">Dries Buytaert</a> (Boston, <a href="https://drupal.org">Drupal</a>)</li>
+              <li><span>$10k</span> <a href="http://assemble.vc">Assemble.vc</a> (Boston)</li>
+              <li><span>$5k</span> <a href="https://www.linkedin.com/in/toonvanagt">Toon Vanagt</a> (Brussels, <a href="http://data.be">data.be</a>)</li>
+              <li><span>$5k</span> <a href="https://www.linkedin.com/in/xaviercorman">Xavier Corman</a> (Brussels, <a href="http://edebex.com">Edebex</a>)</li>
               <li><span>$5k</span> Personal friend</li>
             </ul>
           </p>
 
+          <p>We did a follow up round of $305k in July 2016 (<a href="https://www.ycombinator.com/docs/SAFE_Cap.rtf">SAFE</a>, $8M cap) with:
+            <ul>
+              <li><span>$100k</span> <a href="https://www.linkedin.com/in/ricardo-gorodisch-9b057889">Ricardo Gorodisch</a> (Argentina, President <a href="http://www.fundacionkaleidos.org/">Foundation Kaleidos</a>)</li>
+              <li><span>$50k</span> <a href="https://www.linkedin.com/in/petekoomen">Pete Koomen</a> (SF, Cofounder/CTO <a href="http://optimizely.com">Optimizely</a>)</li>
+              <li><span>$25k</span> <a href="https://www.linkedin.com/in/jpayne">Jim Payne</a> (NYC, Cofounder MoPub, EIR Accel Partners)</li>
+              <li><span>$25k</span> <a href="https://www.linkedin.com/in/caesar-sengupta-2743b">Caesar Sengupta</a> (Bay Area, VP Product Management at Google)</li>
+              <li><span>$25k</span> <a href="https://www.linkedin.com/in/gkgandhi">Gautam Gandhi</a> (India, Head New Business Development India at Google)</li>
```

Comment:
works.

On Fri, Aug 5, 2016 at 12:54 PM Xavier Damman <notifications@github.com>
wrote:

> In frontend/src/containers/Faq.js
> <https://github.com/OpenCollective/opencollective-website/pull/385#discussion_r73724289>
> :
>
> >                <li><span>$5k</span> Personal friend</li>
> >              </ul>
> >            </p>
> >
> > +          <p>We did a follow up round of $305k in July 2016 (<a href="https://www.ycombinator.com/docs/SAFE_Cap.rtf">SAFE</a>, $8M cap) with:
> > +            <ul>
> > +              <li><span>$100k</span> <a href="https://www.linkedin.com/in/ricardo-gorodisch-9b057889">Ricardo Gorodisch</a> (Argentina, President <a href="http://www.fundacionkaleidos.org/">Foundation Kaleidos</a>)</li>
> > +              <li><span>$50k</span> <a href="https://www.linkedin.com/in/petekoomen">Pete Koomen</a> (SF, Cofounder/CTO <a href="http://optimizely.com">Optimizely</a>)</li>
> > +              <li><span>$25k</span> <a href="https://www.linkedin.com/in/jpayne">Jim Payne</a> (NYC, Cofounder MoPub, EIR Accel Partners)</li>
> > +              <li><span>$25k</span> <a href="https://www.linkedin.com/in/caesar-sengupta-2743b">Caesar Sengupta</a> (Bay Area, VP Product Management at Google)</li>
> > +              <li><span>$25k</span> <a href="https://www.linkedin.com/in/gkgandhi">Gautam Gandhi</a> (India, Head New Business Development India at Google)</li>
>
> What about "Entrepreneur. Former Head New Business Development India at
> Google"?
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenCollective/opencollective-website/pull/385/files/cb7f43ce6c135c5d857d7a58c7e85ab5140a73c1#r73724289>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AF0vPbHqwqTPhSAUOJ3mZ8vvFEmJ7OZZks5qc2qpgaJpZM4JdK2V>
> .
>